tests/phpunit/tests/post.php

```diff
@@ -224,9 +350,10 @@ public function test_wp_count_posts_trash_invalidation() {
 		$this->assertNotEquals( $initial_counts->publish, $after_trash_counts->publish );
 	}
 
-	/**
-	 * @ticket 49685
-	 */
+		/**
+		 * @ticket 49685
+		 * @covers ::wp_count_posts
+		 */
```

Comment:
Indentation is wrong here and in the following phpdoc blocks:

```suggestion
	/**
	 * @ticket 49685
	 * @covers ::wp_count_posts
	 */
```

Reply:
My auto-linter has backfired me.